src/state/state.go

```diff
@@ -358,19 +360,8 @@ func (s *State) SetConfigLocalState(local *model.Config, m *manifest.ConfigManif
 	defer s.mutex.Unlock()
 	state.Local = local
 	state.ConfigManifest = m
-
-	s.MarkPathTracked(m.MetaFilePath())
-	s.MarkPathTracked(m.ConfigFilePath())
-
-	// Process transformation's code blocks
-	for _, block := range local.Blocks {
-		s.MarkPathTracked(block.RelativePath())
-		s.MarkPathTracked(block.MetaFilePath())
-		for _, code := range block.Codes {
-			s.MarkPathTracked(code.RelativePath())
-			s.MarkPathTracked(code.MetaFilePath())
-			s.MarkPathTracked(code.CodeFilePath())
-		}
+	for _, path := range m.GetRelatedPaths() {
```

Comment:
Tym ze som pridal `GetRelatedPaths`, ... tj. ze si cestu poznacim uz pri nacitavani, tak je to tu ovela jednoduchsie.